src/components/Callout/Callout.Props.ts

```diff
@@ -81,3 +95,8 @@ export interface ILink {
    */
   url: string;
 }
+
+export enum CalloutBackgroundColor {
+  white,
+  blue
```

Comment:
no colors in enums. Use semantic meanings.

Also, I am confused. Callout is a generic thing, no?